components/banners/HeroBanner.vue

```diff
@@ -1,22 +1,28 @@
 <template>
   <section class="hero-banner">
-    <div class="banner-content">
-      <h1 class="title">
-        {{ title }}
-      </h1>
-      <h3>
-        {{ subtitle }}
-      </h3>
-    </div>
-    <picture>
-      <source :srcset="desktopUrl" media="(min-width: 800px)">
-      <img :src="mobileUrl" :alt="imageAlt">
-    </picture>
+    <banner-content
+                :title="title"
+                :content="subtitle">
```

Comment:
The `content` prop right here should match. I almost thing we build some basic pure bricks pages and then some basic content pages. I'm not sure about that yet.